portfolio/src/main/webapp/script.js

```diff
@@ -138,3 +122,21 @@ function deleteComment(comment) {
 document.addEventListener('DOMContentLoaded', () => {
   document.getElementById('maxNum').addEventListener('click', displayComments);
 });
+
+document.addEventListener('DOMContentLoaded', () => {
+  document.getElementById('mode').addEventListener('click', toggleMode);
+})
+
+function toggleMode() {
+  let element = document.body;
+  element.classList.toggle("dark-mode");
+
+  let anchors = document.getElementsByTagName("a");     
```

Comment:
you can do this in css;


body.dark-mode a {
   color: rgb(..)
}
body a {
    color: white
}